api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1248,9 +1248,78 @@ const (
 // +kubebuilder:validation:Enum=Karpenter
 type Provisioner string
 
+// Configures when and how to scale down cluster nodes.
+type ScaleDownConfig struct {
+	// enabled determines whether the Cluster Autoscaler should scaled down this cluster.
+	//
+	// +kubebuilder:default=Enabled
+	// +kubebuilder:validation:Enum=Enabled;Disabled
+	// +optional
+	Enabled string `json:"enabled,omitempty"`
+
+	// delayAfterAddSeconds sets how long after scale up the scale down evaluation resumes in seconds
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=10000
```

Comment:
This is less than 3 hours, perhaps making it a 24 hours limit would be sensible? It would be 86400